app/api.py

```diff
@@ -58,46 +58,35 @@ async def collections():
 @API.get("/cavisualizer", response_class=HTMLResponse)
 async def computer_assignment_rating_visualizer():
     """Return an HTML table of the computer assignment
-    ratings in the computer assignment collection of the 
+    ratings in the computer assignment collection of the
     selected mongodb database.
     """
     return computer_assignment_visualizer(API.db)
 
 
-@API.post("/update/mentors")
```

Comment:
Why is this being deleted?

Reply:
I have fixed the missing update and delete endpoints! Refactoring will fix the final error.

Reply:
You have some PEP8 issues. Do you use a linter or IDE with automatic linting?

Reply:
As of now, I use Flake8 which I did enable and cleaned up any errors and whitespaces I had missed previously